content/10.blackbox.md

```diff
@@ -6,7 +6,8 @@ Thus, DL models are relatively hard to interpret when compared to other ML model
 
 However, methods such as Autoencoder (AE, **TODO** check if autoencoder is mentioned elsewhere) are DL strategies that can be used for model interpretation.
 AE is a family of unsupervised methods that aim to learn a new (encoded) representation and minimize the error between the new representation and the input data.
-Tan et al., [@doi:10.1142/9789814644730_0014] has uses denoising autoencoders to extract useful genomics features a from breast cancer dataset. Another example of  model interpretation is the Deep Bind [@doi:10.1038/nbt.3300], which predicts DNA and RNA binding sequences potentially involved in regulatory processes.
+Tan et al., [@doi:10.1142/9789814644730_0014] has used denoising autoencoders to summarize key features from genomic datasets.
+Another example of extracting biological insights from deep learning models is provided by the DeepBind database [@doi:10.1038/nbt.3300], which predicts DNA and RNA binding sequences potentially involved in regulatory processes.
 Deep bind used mutation maps to understand the impact of genetic variants of these binding sites ... **TODO**
 
 Unfortunately, the literature on interpretable DL methods is extremely limited. **TODO** what should we suggest here?
```

Comment:
https://arxiv.org/pdf/1802.09791.pdf has some alright references for possible interpretability related methods for DL in bioinformatics (does mention DeepBind too) that might be a useful place to start.